interfaces/acados_matlab_octave/create_consistent_empty_fields.m

```diff
@@ -29,31 +29,29 @@
 
 %
 
-function model = create_consistent_empty_fields(model, opts)
+function model = create_consistent_empty_fields(model)
```

Comment:
Moving this kind of functionality to the template classes is the next step I guess.